<launch>
    <arg name="visualization" default="true"/>
    <arg name="output_path" default="$(find adaptive_viewutility)/../output"/>
    <arg name="max_experiment_duration" default="200"/>
    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>
    <node pkg="airsim_client" type="test_planner_airsim_node" name="test_planner" output="screen" required="true">
        <!-- <param name="file_path" value="/home/jaeyoung/dev/mesh/groundtruth_roi_meshlab.obj"/> -->
        <param name="file_path" value="/home/jaeyoung/dev/mesh/slope_scaled.obj"/>
        <param name="output_directory_path" value="$(arg output_path)"/>
        <param name="num_experiments" value="1"/>
        <param name="snapshot_interval" value="20"/>
        <param name="viewpoint_noise" value="false"/>
        <param name="max_experiment_duration" value="$(arg max_experiment_duration)"/>
        <param name="image_directory" value="$(arg output_path)/images"/>
    </node>
    <!-- <node pkg="rosbag" type="record" name="rosbag_record"
       args="record -o $(find adaptive_viewutility)/../output/rosbag /grid_map"/> -->
    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find adaptive_viewutility)/launch/config_planner.rviz" />
    </group>
</launch>
